### PR TITLE
qemu: add nvdimm support using DT for arm64

### PR DESCRIPTION
**Pull request by edmond-hk**
As Shameer's patch drops the DT mode when add nvdimm support for arm64. This patch
enable it back.
Adopt Shameer's patch [1]

Fixes: #1148
Signed-off-by: Edmond AK Dantes <edmond.dantes.ak47@outlook.com>

[1] https://patchwork.kernel.org/patch/11174981/

@jon @jbryce

### CODE REVIEW

**Comment by jongwu** [comment]
thanks @edmond-hk , this PR depends on https://github.com/kata-containers/runtime/pull/2990
any comments @jodh-intel @devimc

**Comment by edmond-hk** [comment]
hello @jodh-intel @GabyCT, help test this PR, thanks.

**Comment by jongwu** [comment]
Hello @devimc @GabyCT @jodh-intel, help review this PR, thanks!

File: scripts/configure-hypervisor.sh [hunk 1]
```diff
@@ -371,7 +371,9 @@ generate_qemu_options() {
 		qemu_options+=(size:--disable-cloop)
 		qemu_options+=(size:--disable-dmg)
 		qemu_options+=(size:--disable-parallels)
-		qemu_options+=(size:--disable-vxhs)
+		if [ "${qemu_version_major}" -le 5 ] && [ "${qemu_version_minor}" -lt 1 ]; then
```
Comment (jodh-intel) [inline scripts/configure-hypervisor.sh:374]: This won't do what we want if the version is `4.2` for example. I'd use logic like that on #1151 as it's much clearer if you can use a "floating point version number" in the test.

**Review by jodh-intel — CHANGES_REQUESTED** [review]

File: scripts/configure-hypervisor.sh (hunk 1 above)
Comment (edmond-hk) [reply scripts/configure-hypervisor.sh]: good catch, thanks!

**Review by jodh-intel — APPROVED** [review]
Thanks @edmond-hk.

lgtm

**Review by devimc — APPROVED** [review]

**Pull request merged**